packages/sprotty/src/features/move/move.ts

```diff
@@ -189,6 +202,37 @@ export class MoveCommand extends MergeableCommand implements IStoppableCommand {
         });
     }
 
+    // tests if the edge is attached to the moved element directly or to on of their children
+    protected isAttachedEdge(edge: SRoutableElementImpl): boolean {
+        const source = edge.source;
+        const target = edge.target;
+        const checkMovedElementsAndChildren = (sourceOrTarget: SConnectableElementImpl): boolean => {
+            return Array.from(this.resolvedMoves.values()).some(res => {
```

Comment:
This check potentially traverses the whole subgraph of each moved element, which can be very large in some applications. It's much more efficient to do the traversal the other way around: from the edge source or target element, going up the parent chain, until you find an element contained in `resolvedMoves` (or not).